416.patch

```diff
@@ -0,0 +1,62 @@
+From f191bab20fe6c627dd5807c297e56a742c9bee19 Mon Sep 17 00:00:00 2001
+From: Hari Rana <theevilskeleton@riseup.net>
+Date: Sat, 13 May 2023 14:51:27 -0400
+Subject: [PATCH] linux: Prepare for Flatpak
+
+---
+ ...esktop => io.github.jpochyla.Psst.desktop} |  2 +-
+ .pkg/io.github.jpochyla.Psst.metainfo.xml     | 29 +++++++++++++++++++
+ 2 files changed, 30 insertions(+), 1 deletion(-)
+ rename .pkg/{psst.desktop => io.github.jpochyla.Psst.desktop} (86%)
+ create mode 100644 .pkg/io.github.jpochyla.Psst.metainfo.xml
+
+diff --git a/.pkg/psst.desktop b/.pkg/io.github.jpochyla.Psst.desktop
+similarity index 86%
+rename from .pkg/psst.desktop
+rename to .pkg/io.github.jpochyla.Psst.desktop
+index 3179cb1d..9494061f 100755
+--- a/.pkg/psst.desktop
++++ b/.pkg/io.github.jpochyla.Psst.desktop
+@@ -6,4 +6,4 @@ Name=Psst
+ Terminal=false
+ Type=Application
+ Version=1.0
+-Icon=psst.png
+\ No newline at end of file
++Icon=io.github.jpochyla.Psst
+\ No newline at end of file
+diff --git a/.pkg/io.github.jpochyla.Psst.metainfo.xml b/.pkg/io.github.jpochyla.Psst.metainfo.xml
+new file mode 100644
+index 00000000..e1d46e6e
+--- /dev/null
++++ b/.pkg/io.github.jpochyla.Psst.metainfo.xml
+@@ -0,0 +1,29 @@
++<?xml version="1.0" encoding="UTF-8"?>
++<component type="desktop-application">
++  <id>io.github.jpochyla.Psst</id>
++  <url type="homepage">https://github.com/jpochyla/psst</url>
++
++  <name>Psst</name>
++  <summary>Fast Spotify client with native GUI, without Electron, built in Rust.</summary>
++
++  <metadata_license>CC0-1.0</metadata_license>
++  <project_license>MIT</project_license>
++
++  <description>
++    <p>
++      Fast Spotify client with native GUI, without Electron, built in Rust. Very early in development, lacking in features, stability, and general user experience. It is being tested only on Mac so far, but aims for full Windows and Linux support. Contributions welcome!
++    </p>
++  </description>
++
++  <launchable type="desktop-id">io.github.jpochyla.Psst.desktop</launchable>
++  <screenshots>
++    <screenshot type="default">
++      <image>https://raw.githubusercontent.com/jpochyla/psst/master/psst-gui/assets/screenshot.png</image>
++    </screenshot>
++  </screenshots>
++  <content_rating type="oars-1.1"/>
++  <releases>
++    <release version="master" date="2023-05-13"/>
```

Comment:
this is not a version.

Reply:
@hfiguiere In the case of this app we haven't yet versioned it in a semantic way. What would your suggestion be for a rolling version that's built from the main branch?

Reply:
to wait for a release.

Reply:
I'm a maintainer and while we could make a release if absolutely necessary, is there not a path forward to roll up a version update for every new commit (linked with commit hashes?)?

Reply:
Flathub is not to deliver continuous build. There are github actions that do that.

Flathub is to deliver software to users in a form they can use.

Reply:
I only submitted this MR so we can have a base to work with once there is a stable version available